spiceaidocs/docs/reference/spicepod/datasets.md

```diff
@@ -66,42 +62,35 @@ acceleration:
 
 ## `from`
```

Comment:
Do we need to add a section for `#ref`?

Reply:
Yes, filed https://github.com/spiceai/docs/issues/195 to track